File: app/src/main/java/bj/rxjavaexperimentation/model/collectionrelease/Note.java

package bj.rxjavaexperimentation.model.collectionrelease;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Note {

    @SerializedName("field_id")
    @Expose
    private Integer fieldId;
    @SerializedName("value")
    @Expose
    private String value;

    public Integer getFieldId() {
        return fieldId;
    }

    public void setFieldId(Integer fieldId) {
        this.fieldId = fieldId;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }

}

File: app/src/main/java/bj/vinylbrowser/model/collection/Note.kt
package bj.vinylbrowser.model.collection

import android.os.Parcel
import android.os.Parcelable
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class Note(@SerializedName("field_id") val fieldId: Int, val value: String) : Parcelable {

    override fun writeToParcel(dest: Parcel, flags: Int) {
        dest.writeInt(fieldId)
        dest.writeString(value)
    }

    override fun describeContents(): Int {
        return 0
    }
}